flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/CEPITCase.java

```diff
@@ -630,6 +630,102 @@ public String select(Map<String, List<Event>> pattern) {
         assertEquals(expected, resultList);
     }
 
+    @Test
+    public void testNotFollowedByWithIn() throws Exception {
```

Comment:
Could we support the following pattern? If so, it would be great to add an ITCase.
```
Pattern.begin('A').notFollowedBy('B').followedBy('C').times(0, 2).withIn(Time.milliseconds(3))
```